crates/ruma-events/src/live_location.rs

```diff
@@ -0,0 +1,113 @@
+//! Types for extensible live location message events ([MSC3489]).
+//!
+//! [MSC3489]: https://github.com/matrix-org/matrix-spec-proposals/pull/3489
+
+use ruma_common::{MilliSecondsSinceUnixEpoch, OwnedUserId};
+use ruma_macros::EventContent;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    location::{AssetContent, LocationContent},
+    relation::Reference,
+};
+
+/// `BeaconInfoStateEventContent` is a struct that represents the content of a beacon_info state
+/// event. It contains information about a live location sharing event.
+#[derive(Clone, Debug, Deserialize, Serialize, EventContent)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+#[ruma_event(type = "org.matrix.msc3672.beacon_info", alias = "m.beacon", kind = State, state_key_type = OwnedUserId)]
```

Comment:
This seems like a typo, right?

```suggestion
#[ruma_event(type = "org.matrix.msc3672.beacon_info", alias = "m.beacon_info", kind = State, state_key_type = OwnedUserId)]
```

Reply:
Good catch! I will update and comit